src/Mod/Draft/Draft.py

```diff
@@ -1153,22 +1153,28 @@ def makeBlock(objectslist):
         select(obj)
     return obj
 
-def makeArray(baseobject,arg1,arg2,arg3,arg4=None,name="Array"):
+def makeArray(baseobject,arg1,arg2,arg3,arg4=None,arg5=None,arg6=None,name="Array"):
     '''makeArray(object,xvector,yvector,xnum,ynum,[name]) for rectangular array, or
+    makeArray(object,xvector,yvector,zvector,xnum,ynum,znum,[name]) for rectangular array, or
     makeArray(object,center,totalangle,totalnum,[name]) for polar array: Creates an array
     of the given object
     with, in case of rectangular array, xnum of iterations in the x direction
-    at xvector distance between iterations, and same for y direction with yvector
-    and ynum. In case of polar array, center is a vector, totalangle is the angle
-    to cover (in degrees) and totalnum is the number of objects, including the original.
-    The result is a parametric Draft Array.'''
-    if not FreeCAD.ActiveDocument:
-        FreeCAD.Console.PrintError("No active document. Aborting\n")
-        return
```

Comment:
Could you add back these 3 lines? I see no reason to remove them... Thanks!

Reply:
Done.  I don't know how that happened.  Wasn't intended.

It says branch is out of date with base branch.  Should I update branch?

Reply:
Thanks! No, no problem, I'll rebase when committing. Just waiting for the CI tests to finish.